### PR TITLE
Include timeout duration in timeout message

### PR DESCRIPTION
**Pull request by engyebrahim**
fix: https://github.com/microsoft/testfx/issues/2822

### CODE REVIEW

File: test/IntegrationTests/MSTest.Acceptance.IntegrationTests/Properties/launchSettings.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "MSTest.Acceptance.IntegrationTests": {
       "commandName": "Project",
-      "commandLineArgs": "--treenode-filter /*/*/*/**"
+      "commandLineArgs": "--treenode-filter /*/*/InitializeAndCleanupTimeout/**"
```
Comment (Evangelink) [inline test/IntegrationTests/MSTest.Acceptance.IntegrationTests/Properties/launchSettings.json]: Please revert this change

**Review by Evangelink — COMMENTED** [review]
It'd be good to add (or edit) some tests to check the message

**Review by fhnaseer — APPROVED** [review]

**Review by nohwnd — APPROVED** [review]

**Review by Evangelink — APPROVED** [review]
